code/build.R

```diff
@@ -601,7 +608,7 @@ tippecanoe_join = paste(
 )
 responce = system(tippecanoe_join, intern = TRUE)
 if (.Platform$OS.type == "unix") {
-  command_cd = glue::glue("cd outputdata/{output_folder}")
```

Comment:
Why not cd?

Reply:
it will cause this error: > responce = system(command_all, intern = TRUE)
sh: 1: cd: can't cd to outputdata/outputdata/2024-10-01

Reply:
update by command_cd = glue::glue("cd outputdata/{date_folder}")